packages/riipen-ui/src/components/Collapse.jsx

```diff
@@ -0,0 +1,142 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+import { Transition } from "react-transition-group";
+import css from "styled-jsx/css";
+
+import ThemeContext from "../styles/ThemeContext";
+
+const Collapse = props => {
+  const {
+    children,
+    classes,
+    component: Component = "div",
+    in: inProp,
+    ...other
+  } = props;
+
+  const theme = React.useContext(ThemeContext);
+
+  const wrapperRef = React.useRef(null);
+
+  const getLinkedStyles = () => {
+    return css.resolve`
+      .root {
+        height: 0;
+        min-height: 0px;
+        overflow: hidden;
+        transition: ${theme.transitions.create(["height"])};
+      }
+
+      .entered {
+        height: auto;
+        overflow: visible;
+      }
+      .exited {
+        visibility: hidden;
+      }
+    `;
+  };
+
+  const getWrapperSize = () =>
+    wrapperRef.current ? wrapperRef.current.clientHeight : 0;
+
+  const handleEnter = node => {
+    if (wrapperRef.current) {
+      // Set absolute position to get the size of collapsed content
+      wrapperRef.current.style.position = "absolute";
+    }
+
+    node.style.height = "0px";
```

Comment:
Might be able to just be "0"? I forget if you need px for 0 height

Reply:
`"0"` worked :)